src/app/loginComponents/requests/requests.component.ts

```diff
@@ -118,4 +122,31 @@ export class RequestsComponent extends Base implements OnInit {
   rerequestReview(membership: OrganizationUser) {
     this.requestsService.requestRereview(membership.organization.id);
   }
+
+  removeOrganizationDialog(organizationName: string, organizationStatus: string, organizationID: number) {
+    const confirmationDialogData: ConfirmationDialogData = {
+      title: `Are you sure you wish to delete this ${organizationStatus} organization: ${organizationName}?`,
```

Comment:
Sort of a long title for a dialog. I would just say "Delete <org>". Then add it to the message: "Are you sure you want to the delete <org>? All information associated with <org> will be deleted."

Reply:
Title is now 'Delete Organization' and added more info the the message.